doc/source/conf.py

```diff
@@ -126,6 +126,7 @@
     },
     # Remove sphinx_gallery_thumbnail_number from generated files
     "remove_config_comments": True,
+    "parallel": True,  # Same number of jobs as Sphinx flag `-j`
```

Comment:
Not sure what this comment means.

Reply:
I tried clarifying the comment in https://github.com/scikit-image/scikit-image/pull/7579/commits/09b24f5ccefc0e88d392c685d8e33a8f9776fa29.

> `True` defaults to the number of jobs used by Sphinx (see its flag `-j`)